src/config.ts

```diff
@@ -127,3 +127,6 @@ export const deploys = {
 
 const stableDeploys = [...deploys.production, deploys.staging]
 export const isStableDeploy = stableDeploys.some(url => window.location.origin === url)
+
+export const customAppTitle = process.env.REACT_APP_TITLE
```

Comment:
Functions like getAppTitle would be useful so components don't reimplement checks

Reply:
Would you prefer having a function instead of a const? (I don't think this can change without reloading the app.)

Reply:
OK, introduced `getAppTitle()`.